README.md

```diff
@@ -19,6 +19,10 @@ Dynamatte: A dynamic matting method to generate in scene video fusion
 +Achleschwar
 $Chris
 
+## SAM Installation
+```
+pip install git+https://github.com/facebookresearch/segment-anything.git
```

Comment:
Good to add. Added this to the requirements.txt file as well so may be redundant

Reply:
yup agreed